index.js

```diff
@@ -17,7 +17,8 @@ const {
 const max = Math.max;
 
 const defaultOptions = {
-  includeResourcesFromDiskCache: false
+  includeResourcesFromDiskCache: false,
+  splitPagesWithinFrame: false
```

Comment:
Could you explain in more detail what you'd like this to do? You don't create new pages for sub frames (iframes), so it's not strictly one page per loaded url. A test case would be enlightening (add an options parameter to parsePerflog in tests.js, so tests can specify how to parse). I tried, but couldn't get different output based on this option.